src/openrct2/actions/TrackDesignAction.h

```diff
@@ -50,6 +50,11 @@ DEFINE_GAME_ACTION(TrackDesignAction, GAME_COMMAND_PLACE_TRACK_DESIGN, TrackDesi
     {
     }
 
+    void AcceptParameters(GameActionParameterVisitor & visitor) override
+    {
+        visitor.Visit(_loc);
+    }
```

Comment:
Doesn't `TrackDesign` need to be visited as well?

Reply:
At some point, I am not implementing it for plugins right now as it is very complicated.

Reply:
Ok, since all of the others visit every member, I think we can leave a TODO here to make it clear it's intentionally unimplemented

Reply:
Done